examples/evaluate_lightning_text_classification.py

```diff
@@ -1,42 +1,36 @@
 import pprint
 from pathlib import Path
 
+from pytorch_lightning.callbacks import EarlyStopping
+
 from embeddings.defaults import RESULTS_PATH
 from embeddings.pipeline.lightning_classification import LightningClassificationPipeline
 
 root: Path = RESULTS_PATH.joinpath("lightning_sequence_classification")
 root.mkdir(parents=True, exist_ok=True)
 
+
 pipeline = LightningClassificationPipeline(
     embedding_name="allegro/herbert-base-cased",
```

Comment:
I would unify this example to other's. Add parameters like in the flair example. And unify naming, we use `document_classification` name in flair based example.